### ISSUE TITLE
Error loading more than one ddimageortext question in the same pare

### ISSUE THREAD

**Issue by crazyserver** [moodleou/moodle-qtype_ddimageortext#27]
JS console has the next error:
```
Uncaught ReferenceError: pendingid is not definedyui_combo.php?m/-1/qtype_ddimageortext/dd/dd-debug.js:263 Y.extend.initializer
```
This causes that the second question is not loaded and cannot be answered.

I've found that the line described:
```js
Y.later(500, this, this.reposition_drags_for_question, [pendingid], true);
```

Has to be replace by
```js
Y.later(500, this, this.reposition_drags_for_question, [this.pendingid], true);
```

**Comment by timhunt** [comment]
The fix for #28 fixed this too. Closing.

Note, there was a similar bug (thought it did not seem to cause any bad symptoms) in ddmarker, which I have just fixed.

**Comment by CanadianEngineer** [comment]
I'm having the same issue as you guys had, where the drag to text objects were being shifted. I was wondering which file did you modify in order to fix this issue? I can't seem to find it.
